vassal-app/src/main/java/VASSAL/build/module/properties/PropertyPrompt.java

```diff
@@ -36,6 +36,10 @@ public PropertyPrompt(Constraints constraints, String prompt) {
     this.promptText = prompt;
   }
 
+  public Constraints getConstraints() {
+    return constraints;
+  }
+
```

Comment:
?

Reply:
Weird. Took it out. Nothing bad happened.

Reply:
I probably thought I needed it for something at one point and then change my mind but forgot to take it back out.